(GH 33) Create a room at grid coordinates x:6, y:4

danyelcampbell opened the pull request:
added room x6:y4
also updated the map to reflect the newly added room

danyel-campbell committed:
(GH 33) Create a room at grid coordinates x:6, y:4

added room x6:y4
also updated the map to reflect the newly added room

dalath commented:
I'm so sorry, I think I forgot to close the 6:4 room issue. Someone else put a Panic Room there already. Would it be ok for you to move your Champagne Room to a different coordinate?

danyelcampbell commented:
I can move it, no problem. What would be a good location?

danyel-campbell committed:
(GH 53) - Create a room at grid coordinates x:6, y:5

Changed Champagne Room from room x:6 y:4 to room x:6 y:5 on main.js and on MAP.txt

Commit:
Merge branch 'master' into master

dalath commented:
Good job!
